doc/content/getting_started/installation.md

```diff
@@ -0,0 +1,74 @@
+# Install Moltres
+
+## 1. Install the Conda MOOSE Environment
+
+Moltres relies on the MOOSE framework. We suggest that users install the MOOSE environment using
+Conda packages by following the instructions in the "Install Mambaforge3" and "Install MOOSE Conda
+Packages" section of the MOOSE
+[installation guide](https://mooseframework.inl.gov/getting_started/installation/conda.html). The
+`moose-tools` and `moose-libmesh` Conda package versions which are compatible with the current
+MOOSE Git submodule on this repository are `moose-tools=2021.07.14` and `moose-libmesh=2021.07.14`.
+We cannot guarantee compatibility with the older/newer MOOSE builds and their corresponding Conda
+environments. Run the following command to install the specified package versions:
+
+```bash
+mamba install moose-tools=2021.07.14 moose-libmesh=2021.07.14
+```
+
+## 2. Clone Moltres
```

Comment:
Maybe we should add an alert window in the "contributing to moltres" page that links to here so they can see how the steps to build / install their fork when they're developing might differ. What do you think?

Reply:
I think that's a good idea! I still sometimes forget to recompile Moltres after making changes to the source code.